pkgs/applications/networking/remote/citrix-workspace/generic.nix

```diff
@@ -102,6 +102,7 @@ stdenv.mkDerivation rec {
 
   runtimeDependencies = [
     glib
+    glib-networking
```

Comment:
please add this to the parameters list at the top of the file.

Reply:
Will do.